### PR TITLE
[Windows] Make WebCore to be compiled with MSVC on Windows

### PR DESCRIPTION
**Pull request by vitallium**

### CODE REVIEW

File: Source/WebCore/WebCorePrefix.h [hunk 1]
```diff
@@ -100,6 +100,7 @@
 #include <sys/resource.h>
 #endif
 
+#if PLATFORM(QT) && !OS(WINDOWS)
```
Comment (annulen) [inline Source/WebCore/WebCorePrefix.h]: I believe #if USE(CF) should be used here

File: Source/WebCore/bindings/js/JSDOMWindowBase.h [hunk 2]
```diff
@@ -20,6 +20,7 @@
 #ifndef JSDOMWindowBase_h
 #define JSDOMWindowBase_h
 
+#include "DOMWindow.h"
```
Comment (annulen) [inline Source/WebCore/bindings/js/JSDOMWindowBase.h]: Why this change is necessary?

Comment (Vitallium) [inline Source/WebCore/bindings/js/JSDOMWindowBase.h]: MSVC requires the type to be fully defined before referencing it.

**Comment by annulen** [comment]
It would be great if you upstreamed MSVC-related changes
